packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -600,6 +600,45 @@ export class LocalCorpus<
   }
 }
 
+/** A fully mananged {@link Corpus} served by Fixie. */
+export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implements Corpus<ChunkMetadata> {
+  private static readonly FIXIE_API_URL = 'https://app.fixie.ai/api';
```

Comment:
I think we'll want to make this readable via env var as well so people can proxy it if they want, just like they proxy OpenAI calls in [Arch 2](https://docs.ai-jsx.com/guides/architecture?_highlight=arch#ui--aijsx-on-the-client-api-calls-on-the-server).

Reply:
See also https://docs.ai-jsx.com/guides/openai#set-a-proxy-env-var.